plugins/outputs/execd/README.md

```diff
@@ -34,8 +34,14 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
   ## Flag to determine whether execd should throw error when part of metrics is unserializable
   ## Setting this to true will skip the unserializable metrics and process the rest of metrics
   ## Setting this to false will throw error when encountering unserializable metrics and none will be processed
+  ## This setting is ignored if use_batch_format is true.
   # ignore_serialization_error = false
 
+  ## Use batch serialization format instead of line based delimiting.  The
```

Comment:
I know we talk about "lines of line protocol", but I would say individual metrics instead of line-based. Same below.

Reply:
I copied this from file output plugin. I agree though, I'll fix that up.